pgt/src/PGT/CLI.hs

```diff
@@ -38,10 +38,11 @@ parserInfo = wrapHelper subcommands
 
     subcommands =
       CLI.subparser
-        $  mkCommand "list"   runList
-        <> mkCommand "run"    runExamples
-        <> mkCommand "test"   runTests
-        <> mkCommand "update" runUpdates
+        $  mkCommand "list"      runList
+        <> mkCommand "run"       runExamples
+        <> mkCommand "test"      runTests
+        <> mkCommand "update"    runUpdates
+        <> mkCommand "update-v2" runUpdatesWithLinter
```

Comment:
Comment: I also considered `update-with-linter` for the command line name. The reasoning behind the `v2` suffice is communicating the intent of replacing the current `update` command with the new one at some point.

Reply:
__comment:__ I think either `update-v2` or `update-with-linter` is fine. Given it's only temporary, I don't think it matter which you choose.